src/lib/lan7430conf/CMakeLists.txt

```
@@ -72,3 +72,9 @@ install(TARGETS ${PROJECT_NAME}
     ARCHIVE DESTINATION ${_lib}
     PUBLIC_HEADER DESTINATION ${_include}/lan7430conf
 )
+
+install(EXPORT
```

```suggestion
install(EXPORT  ${PROJECT_CMAKE_EXPORT_NAME}
```
Maybe putting everything in one line like the other statements is more consistent across the file